weaviate/collections/cluster/cluster.py

```diff
@@ -73,6 +71,17 @@ async def nodes(
             `weaviate.EmptyResponseError`
                 If the response is empty.
         """
+        nodes = await self.rest_nodes(collection, output)
+        if output == "verbose":
+            return _ConvertFromREST.nodes_verbose(nodes)
+        else:
+            return _ConvertFromREST.nodes_minimal(nodes)
+
+    async def rest_nodes(
+        self,
+        collection: Optional[str] = None,
+        output: Optional[Literal["minimal", "verbose"]] = None,
+    ) -> List[NodeREST]:
```

Comment:
Since you are adding this method to `_ClusterAsync`, you also need to add its synchronous equivalent signature to the sync stubs in `collections/cluster/sync.pyi` alongside the stubs for the `def nodes(...)` method, cheers!

Reply:
Done as well